From 945beaa1154af8cacf6d5663aec25ab3707bbb81 Mon Sep 17 00:00:00 2001
From: Daniel Scherzer <daniel@wikiteq.com>
Date: Sat, 1 Nov 2025 20:15:06 -0700
Subject: [PATCH] `ext.swl.watchlistconditions`: remove `es6-polyfills`
 dependency

Fixes #122
---
 extension.json | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/extension.json b/extension.json
index 8a9f700..4e4f8df 100644
--- a/extension.json
+++ b/extension.json
@@ -143,8 +143,7 @@
 				"swl-custom-input"
 			],
 			"dependencies": [
-				"mediawiki.api",
-				"es6-polyfills"
+				"mediawiki.api"
 			]
 		}
 	},
